### DISCUSSION TITLE
pyload pwa

### DISCUSSION THREAD

**Discussion by StefftheEmperor2** [pyload/pyload#4249] [Help]
Hi, last week i had covid and plenty of time. so a started a progressive web app for pyload . is anyone in to support development? i'm currently using react native web / expo. the connection made to pyload backend is realized using websockets. originally i was looking forward to use wamp, but started to realize it would need a lot of middleware for just some little tasks.

**Comment by milahu** [comment]
similar ptojects
https://github.com/ziahamza/webui-aria2
https://github.com/binux/yaaw
https://github.com/JeelPatel231/aria2-web-interface

see also https://github.com/pyload/pyload/issues/4414#issuecomment-2806297403
